### DISCUSSION TITLE
The code of webpack are not use babel.

### DISCUSSION THREAD

**Discussion by nonusx** [babel/babel#12831] [Q&A] [answered]
I'm used webpack and babel(include preset-env). My code was successfully converted by babel. But the module code automatically generated by webpack does not use babel.

Like this:
```
(()=>{//myCode})()//webpackCode
```

The arrow function of es6 is used in the outermost IIFE.

I want it to be a function expression of ES5.

**Comment by nicolo-ribaudo** [comment, marked as answer]
You need https://webpack.js.org/configuration/output/#outputenvironment.
